docs/tutorial/index.rst

```diff
@@ -299,7 +299,7 @@ This has several advantages:
 - Some configurations are only available using the config file.
 
 Read the Docs works without this configuration
-by :doc:`making some decisions on your behalf </default-versions>`.
+by :ref:`making some decisions on your behalf <default-versions>`.
```

Comment:
You can use `` :ref:`builds:Default versions of dependencies` `` instead of hardcoding the ref.

Reply:
Woops, I'm not used to have section autolabeling. thanks